src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,27 @@ public function __toString()
     {
         return (string) $this->value;
     }
+
+    /**
+     * Creates ConstStub from bitfield flag.
+     *
+     * @param int    $value  The value of bitfield flag
+     * @param string $prefix The prefix filter applied on constant names
+     *
+     * @return self
+     */
+    public static function fromFlag($value, $prefix)
```

Comment:
What about https://3v4l.org/EBmJg (L14 changed)?

So yes, we dump ``E_WARNING | E_NOTICE`` and we have ``E_WARNING | E_SOMETHING_UNRELATED | E_NOTICE`` as result with no wanted ``E_SOMETHING_UNRELATED``. But ``E_WARNING | E_NOTICE === E_WARNING | E_SOMETHING_UNRELATED | E_NOTICE`` is true (``(2 | 8) === (2 | 2 | 8)``). More, by showing ``E_SOMETHING_UNRELATED`` in result, we can understand that we have misnamed this constant. This issue is *partially* solved no?